docs/developer-guide/contributing.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to zot Development
+
+> :point_right: The zot project is built for developers by developers. The zot project welcones the participation of the open source community in extending and improving zot. 
+
+## Submission Requirements
+
+> Summary: All contributions must meet these requirements:
+> 
+> - Adhere to the Apache license
+> - Be submitted by a pull request (PR) from your fork
+> - Commits must have a 
+> 
+### License
+
+zot is released under the [Apache License 2.0](https://github.com/project-zot/zot/blob/main/LICENSE). All contributions must adhere to this license.
+
```

Comment:
Include the following text either in your commit msg or in the PR description.

"By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license."